.releaserc

```diff
@@ -0,0 +1,54 @@
+{
+  "branches": [
+    "main",
+    {
+      "name": "dev",
+      "prerelease": true
+    }
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "conventionalCommits",
+        "releaseRules": [
+          { "type": "style", "release": "patch" },
+          { "type": "refactor", "release": "patch" },
+          { "type": "perf", "release": "patch" },
+          { "type": "BREAKING", "release": "major" },
+          { "type": "BREAKING CHANGE", "release": "major" }

```

Comment:
Are these really required? Why not go with the default rules?

Reply:
Default rules require putting `BREAKING CHANGE` in the footer instead of directly in the commit title, Other types are also added because they're usually used when developing an app but it's not really required. Should I remove this?

Reply:
`BREAKING CHANGE` does sound like it belongs in the footer to me, ! is used to represent breaking changes in the commit title when following conventional commit schemes

Reply:
> `BREAKING CHANGE` does sound like it belongs in the footer to me, ! is used to represent breaking changes in the commit title when following conventional commit schemes

I'll remove the `BREAKING CHANGE` ones, what about the other ones I added?